pennylane/ops/op_math/controlled.py

```diff
@@ -109,7 +111,7 @@ def workflow(theta, w, cw):
             def func(arg):
               qml.RX(theta, wires=arg)
 
-            @qml.cond(theta > 0.0)
+            @catalyst.cond(theta > 0.0)
```

Comment:
has the catalyst team confirmed that we're ok with advertising it this way? I know you spotted an issue using `qml.cond`, so I just wanna make sure

Reply:
Yes. This is the one way to get this code to work, as `catalyst.cond`, `qml.cond` + qjit, and `qml.cond` all have different signatures and behaviours. 

We could probably do a bigger rewrite to this example instead, but this more or less works.

Reply:
@maliasadi

Reply:
change reverted, as is fixed in PR #5006